pages/dao/[symbol]/proposal/components/instructions/raydium/AddLiquidity.tsx

```diff
@@ -111,27 +103,23 @@ const AddLiquidityRaydium = ({
       form.governedAccount?.governance?.account &&
       wallet?.publicKey
     ) {
-      const tokenA = getGovernanceToken(connection.cluster, form.tokenAName)
-      const tokenB = getGovernanceToken(connection.cluster, form.tokenBName)
-
+      const poolKeys = getLiquidityPoolKeysByLabel(form.liquidityPool)
+      const base = await connection.current.getTokenSupply(poolKeys.baseMint)
```

Comment:
it may be interesting to parallelise here: 

```
const [
  base,
  quote,
] = await Promise.all([
   connection.current.getTokenSupply(poolKeys.baseMint),
   connection.current.getTokenSupply(poolKeys.quoteMint),
]);
```

Reply:
I remember you told me that indeed!